tests/SprykerEcoTest/Zed/PunchoutCatalogs/Business/PunchoutCatalogsFacadeTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerTest\Zed\PunchoutCatalogs\Business;
+
+use Codeception\Test\Unit;
+use Generated\Shared\Transfer\CompanyBusinessUnitTransfer;
+use Generated\Shared\Transfer\CompanyTransfer;
+use Generated\Shared\Transfer\CompanyUserTransfer;
+use Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer;
+use Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnection;
+use SprykerEco\Zed\PunchoutCatalogs\Business\PunchoutCatalogsFacadeInterface;
+
+/**
+ * Auto-generated group annotations
+ * @group SprykerTest
+ * @group Zed
+ * @group PunchoutCatalogs
+ * @group Business
+ * @group Facade
+ * @group PunchoutCatalogsFacadeTest
+ * Add your own group annotations below this line
+ */
+class PunchoutCatalogsFacadeTest extends Unit
+{
+    protected const NOT_EXISTING_CONNECTION_ID = '21';
+    protected const CONNECTION_NAME = 'Test name';
+    protected const CONNECTION_USERNAME = 'Test username';
+    protected const CONNECTION_TYPE = 'Test type';
+    protected const CONNECTION_FORMAT = 'Test format';
+
+    /**
+     * @var \SprykerEcoTest\Zed\PunchoutCatalogs\PunchoutCatalogsBusinessTester
+     */
+    protected $tester;
+
+    /**
+     * @return void
+     */
+    public function testFindConnectionByIdReturnsNullIfNothingFound(): void
+    {
+        // Act
+        $punchoutConnectionTransfer = $this->getFacade()
+            ->findConnectionById(static::NOT_EXISTING_CONNECTION_ID);
+
+        // Assert
+        $this->assertNull($punchoutConnectionTransfer);
+    }
+
+    /**
+     * @return void
+     */
+    public function testFindConnectionByIdFindsConnectionById(): void
```

Comment:
`testFindConnectionByIdRetrievesConnectionWhenProvidedIdExists`.

Reply:
Renamed to `testFindConnectionByIdRetrievesConnectionWhenItExists`